docs/F-Refinements/two-dimensional-arrays.md

```diff
@@ -267,22 +269,23 @@ The above example references the address of the third character within the secon
 
 ### Input and Output
 
-**Input**
+* **Input**
 
 To accept input for a list of 5 names, we write:
 
 ```c
 int i;
 char name[5][31];
 
-for (i = 0; i < 5; i++)
-        scanf(" %[^\n]", name[i]); 
+for (i = 0; i < 5; i++){
+    scanf(" %[^\n]", name[i]); 
+}
 ```
 
-> **Reminder**<br/>
+> **Reminder**
```

Comment:
Suggestion to add an admonition. For example, you may use the following -> `:::note` or `:::info`

Reply:
Great! That will make the md file looks nicer!